org.ghidra_sre.Ghidra.json

```diff
@@ -34,56 +34,73 @@
+                "cp -r ghidra_*_FLATPAK /app/ghidra",
+                "mkdir -p /app/share/applications",
+                "cp org.ghidra_sre.Ghidra.desktop /app/share/applications",
+                "mkdir -p /app/share/icons",
+                "cp ghidra_8_256x256x32.png /app/share/icons/org.ghidra_sre.Ghidra.png",
+                "mkdir -p /app/share/appdata",
+                "cp org.ghidra_sre.Ghidra.appdata.xml /app/share/appdata",
+                "ln -s /app/ghidra/ghidraRun /app/bin/ghidra"
             ],
+            "build-options": {
+                "build-args": [
+                    "--share=network"
+                ]
+            },
             "sources": [
                 {
-                    "type": "archive",
-                    "sha256": "c1f18cdb12e2e1c0313e7becf7f0387226937ac67ad6c6e5056fa889229f969a",
-                    "url": "https://ghidra-sre.org/ghidra_9.2.4_PUBLIC_20210427.zip",
+                    "type": "git",
+                    "commit": "63bb30ac8bee0e3ff34c9812d751edcfdc70dbe4",
+                    "tag": "Ghidra_10.1.2_build",
+                    "url": "https://github.com/NationalSecurityAgency/ghidra.git",
                     "x-checker-data": {
-                        "type": "json",
-                        "url": "https://api.github.com/repos/NationalSecurityAgency/ghidra/releases/latest",
-                        "version-query": ".tag_name | sub(\"^Ghidra_\"; \"\") | sub(\"_build\"; \"\")",
-                        "url-query": ".assets[] | .browser_download_url"
+                        "type": "git",
+                        "tag-pattern": "^Ghidra_([\\d.]+)_build$"
                     }
                 },
+                {
+                    "type": "patch",
+                    "paths": [
+                        "app_name.patch",
+                        "last_modified.patch"
+                    ]
+                },
                 {
                     "type": "file",
                     "path": "org.ghidra_sre.Ghidra.desktop"
                 },
                 {
                     "type": "file",
-                    "path": "org.ghidra_sre.Ghidra.metainfo.xml"
+                    "path": "org.ghidra_sre.Ghidra.appdata.xml"
                 },
                 {
                     "type": "script",
-                    "dest-filename": "ghidra.sh",
+                    "dest-filename": "build.sh",
                     "commands": [
-                        "# Create a default ghidra.properties if one wasn't already set, for the user to edit",
-                        "[[ -f /var/config/ghidra.properties ]] || cp /app/lib/ghidra/support/launch.properties.orig /var/config/ghidra.properties",
-                        "exec /app/lib/ghidra/ghidraRun \"$@\""
+                        "source /usr/lib/sdk/openjdk11/enable.sh",
+                        "gradle -I gradle/support/fetchDependencies.gradle init",
```

Comment:
I can take bets that this isn't going to build. There's no network support inside the flatpak-builder, the dependencies need to be fetched ahead of time.

Reply:
Line `65` is solving the network issue